ios/FluentUI/Core/Colors.swift

```diff
@@ -281,67 +281,150 @@ public final class Colors: NSObject {
 
     /// Use these funcs to grab a color customized by a ColorProviding object for a specific window.. If no colorProvider exists for the window, falls back to deprecated singleton theme color
     @objc public static func primary(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryColor(for: window) ?? primary
+        return colorProvidersMap.object(forKey: window)?.primaryColor(for: window) ?? FallbackThemeColor.primary
     }
 
     @objc public static func primaryTint10(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryTint10Color(for: window) ?? primaryTint10
+        return colorProvidersMap.object(forKey: window)?.primaryTint10Color(for: window) ?? FallbackThemeColor.primaryTint10
     }
 
     @objc public static func primaryTint20(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryTint20Color(for: window) ?? primaryTint20
+        return colorProvidersMap.object(forKey: window)?.primaryTint20Color(for: window) ?? FallbackThemeColor.primaryTint20
     }
 
     @objc public static func primaryTint30(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryTint30Color(for: window) ?? primaryTint30
+        return colorProvidersMap.object(forKey: window)?.primaryTint30Color(for: window) ?? FallbackThemeColor.primaryTint30
     }
 
     @objc public static func primaryTint40(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryTint40Color(for: window) ?? primaryTint40
+        return colorProvidersMap.object(forKey: window)?.primaryTint40Color(for: window) ?? FallbackThemeColor.primaryTint40
     }
 
     @objc public static func primaryShade10(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryShade10Color(for: window) ?? primaryShade10
+        return colorProvidersMap.object(forKey: window)?.primaryShade10Color(for: window) ?? FallbackThemeColor.primaryShade10
     }
 
     @objc public static func primaryShade20(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryShade20Color(for: window) ?? primaryShade20
+        return colorProvidersMap.object(forKey: window)?.primaryShade20Color(for: window) ?? FallbackThemeColor.primaryShade20
     }
 
     @objc public static func primaryShade30(for window: UIWindow) -> UIColor {
-        return colorProvidersMap.object(forKey: window)?.primaryShade30Color(for: window) ?? primaryShade30
+        return colorProvidersMap.object(forKey: window)?.primaryShade30Color(for: window) ?? FallbackThemeColor.primaryShade30
     }
 
     private static var colorProvidersMap = NSMapTable<UIWindow, ColorProviding>(keyOptions: .weakMemory, valueOptions: .weakMemory)
 
+    fileprivate enum FallbackThemeColor {
```

Comment:
Why creating an enum these properties?
It's confusing to have a enum with no enumeration values.

Reply:
It's used as an [uninhabitable type](https://nshipster.com/never/), which means this type act as only a namespace. Actually Apple uses this technique a lot in Swift repo (https://github.com/apple/swift/blob/a4230ab2ad37e37edc9ed86cd1510b7c016a769d/stdlib/public/core/Unicode.swift#L918)

Reply:
Just added some comment to make my intention clear :)